### PR TITLE
i18zh-cn: Translation of /guides/index.mdx

### PR DESCRIPTION
**Pull request by hangriver**
#### What kind of changes does this PR include?

- Translated content for Tauri 2.0 docs

#### Description
- This PR offers a zh-cn translation for [What is Tauri?](https://github.com/tauri-apps/tauri-docs/blob/next/src/content/docs/guides/index.mdx) page.

Files added are:
- /zh-CN/guides/index.mdx
- /zh-CN/concepts/rendering.mdx
- /zh-CN/concepts/size.mdx

PS. This is my first time helping translate docs, so I apologize if any mistakes occur.

### CODE REVIEW

**Comment by lorenzolewis** [comment]
Awesome, thank you @hangriver! The two pages under concepts aren't ready for translation yet since they're just stubs. Could you remove those? You can use the dashboard to see which pages are ready for translation here: https://beta.tauri.app/contribute/translate-status

@dklassic would you be able to help out with a review on this?

Sidenote: I'm seeing the dashboard appears to be a tad bit broken for `zh-CN`, I wonder if that's because it's case-sensitive and looking for `zH-cn` instead of `zh-CN`? 🤔

File: src/content/docs/zh-CN/guides/index.mdx [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+title: Tauri是什么？
+i18nReady: true
+---
+
+:::caution[先行版软件]
+
+这份文档是为Tauri 2.0先行版编撰的，将会随着开发进程动态调整。如果你希望查看Tauri 1.0版本文档，请访问：https://tauri.app 。
```
Comment (dklassic) [inline src/content/docs/zh-CN/guides/index.mdx]:
```suggestion
这份文档是为 Tauri 2.0 先行版编撰的，将会随着开发进程动态调整。如果你希望查看 Tauri 1.0 版本文档，请访问：https://tauri.app 。
```
Improve readability.

File: src/content/docs/zh-CN/guides/index.mdx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+:::
+
+Tauri是一个构建适用于所有主流桌面和移动平台的轻快二进制文件的框架。 开发者们可以集成任何用于创建用户界面的可以被编译成HTML、JavaScript和CSS的前端框架，同时可以在必要时使用Rust、Swift和Kotlin等语言编写后端逻辑。
```
Comment (dklassic) [inline src/content/docs/zh-CN/guides/index.mdx]:
```suggestion
Tauri 是一个构建适用于所有主流桌面和移动平台的轻快二进制文件的框架。开发者们可以集成任何用于创建用户界面的可以被编译成 HTML、JavaScript 和 CSS 的前端框架，同时可以在必要时使用 Rust、Swift 和 Kotlin 等语言编写后端逻辑。
```
Improve readability and remove excessive space.

File: src/content/docs/zh-CN/guides/index.mdx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+你可以使用下面的任意一个命令以利用 [`create-tauri-app`](https://github.com/tauri-apps/create-tauri-app) 创建一个新的项目. 请务必参考 [前置要求文档](/zh-cn/guides/prerequisites/) 安装所有Tauri必须的依赖，并阅读 [前端配置指南](/zh-cn/guides/frontend/) 了解推荐的前端配置方案。
```
Comment (dklassic) [inline src/content/docs/zh-CN/guides/index.mdx]:
```suggestion
你可以使用下面的任意一个命令以利用 [`create-tauri-app`](https://github.com/tauri-apps/create-tauri-app) 创建一个新的项目。请务必参考[前置要求文档](/zh-cn/guides/prerequisites/)安装所有 Tauri 必须的依赖，并阅读[前端配置指南](/zh-cn/guides/frontend/)了解推荐的前端配置方案。
```
Improve readability, remove excessive space, and fix punctuation.

File: src/content/docs/zh-CN/guides/index.mdx [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+import Cta from '../../_fragments/cta.mdx';
+
+<Cta />
+
+在你成功创建了你的第一个应用后，你可以在[功能及秘诀列表](/zh-cn/features/)探索Tauri的不同功能及秘诀。
```
Comment (dklassic) [inline src/content/docs/zh-CN/guides/index.mdx]:
```suggestion
在你成功创建了你的第一个应用后，你可以在[功能及秘诀列表](/zh-cn/features/)探索 Tauri 的不同功能及秘诀。
```
Improve readability.

File: src/content/docs/zh-CN/guides/index.mdx [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+## 为什么选用Tauri？
```
Comment (dklassic) [inline src/content/docs/zh-CN/guides/index.mdx]:
```suggestion
## 为什么选用 Tauri？
```
Improve readability.

File: src/content/docs/zh-CN/guides/index.mdx [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+
+对于开发者而言，Tauri有三个主要优势：
```
Comment (dklassic) [inline src/content/docs/zh-CN/guides/index.mdx]:
```suggestion
对于开发者而言，Tauri 有三个主要优势：
```
Improve readability.

File: src/content/docs/zh-CN/guides/index.mdx [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+
+- 构建应用所需的可靠基础
+- 使用系统原生webview（网页视图）带来的更小打包体积
```
Comment (dklassic) [inline src/content/docs/zh-CN/guides/index.mdx]:
```suggestion
- 使用系统原生 webview（网页视图）带来的更小打包体积
```
Improve readability.

File: src/content/docs/zh-CN/guides/index.mdx [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+- 使用任何前端技术和多种语言绑定带来的灵活性
+
+阅读[Tauri 1.0博客文章](/blog/tauri-1-0)深入了解Tauri哲学思想。
```
Comment (dklassic) [inline src/content/docs/zh-CN/guides/index.mdx]:
```suggestion
阅读[Tauri 1.0 博客文章](/blog/tauri-1-0)深入了解 Tauri 哲学思想。
```
Improve readability.

File: src/content/docs/zh-CN/guides/index.mdx [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+
+### 可靠的基础
+
+由于Tauri是使用Rust构建的，它可以利用Rust提供的内存、线程和类型安全方面的优势。而使用Tauri开发的应用也可以从中受益，甚至无需由Rust专家开发这些应用。
```
Comment (dklassic) [inline src/content/docs/zh-CN/guides/index.mdx]:
```suggestion
由于 Tauri 是使用 Rust 构建的，它可以利用 Rust 提供的内存、线程和类型安全方面的优势。而使用 Tauri 开发的应用也可以从中受益，甚至无需由 Rust 专家开发这些应用。
```
Improve readability.

File: src/content/docs/zh-CN/guides/index.mdx [hunk 10]
```diff
@@ -0,0 +1,54 @@
...
+
+Tauri的每一个主要和次要版本都会接受安全审计。这一审计不光涵盖了Tauri组织范围内的代码，还包括了Tauri所使用的上游依赖。当然，这些举措不能消除所有的风险，但它们能够为开发者提供一个坚实的基础。
```
Comment (dklassic) [inline src/content/docs/zh-CN/guides/index.mdx]:
```suggestion
Tauri 的每一个主要和次要版本都会接受安全审计。这一审计不光涵盖了 Tauri 组织范围内的代码，还包括了 Tauri 所使用的上游依赖。当然，这些举措不能消除所有的风险，但它们能够为开发者提供一个坚实的基础。
```
Improve readability.

File: src/content/docs/zh-CN/guides/index.mdx [hunk 11]
```diff
@@ -0,0 +1,54 @@
...
+
+阅读[Tauri安全策略](https://github.com/tauri-apps/tauri/security/policy)和[Tauri 1.0审计报告](https://github.com/tauri-apps/tauri/blob/dev/audits/Radically_Open_Security-v1-report.pdf)了解更多信息。
```
Comment (dklassic) [inline src/content/docs/zh-CN/guides/index.mdx]:
```suggestion
阅读 [Tauri 安全策略](https://github.com/tauri-apps/tauri/security/policy)和 [Tauri 1.0 审计报告](https://github.com/tauri-apps/tauri/blob/dev/audits/Radically_Open_Security-v1-report.pdf)了解更多信息。
```
Improve readability.

File: src/content/docs/zh-CN/guides/index.mdx [hunk 12]
```diff
@@ -0,0 +1,54 @@
...
+
+### 更小的体积
+
+Tauri利用了已经存在于每一个用户系统的webview。Tauri应用中只包含了该应用专属的代码和资源文件，不需要在每个应用中都打包一个浏览器引擎，这意味着一个最小化的Tauri应用体积可能小于600KB。
```
Comment (dklassic) [inline src/content/docs/zh-CN/guides/index.mdx]:
```suggestion
Tauri 利用了已经存在于每一个用户系统的 webview。Tauri 应用中只包含了该应用专属的代码和资源文件，不需要在每个应用中都打包一个浏览器引擎，这意味着一个最小化的 Tauri 应用体积可能小于 600KB。
```
Improve readability.

File: src/content/docs/zh-CN/guides/index.mdx [hunk 13]
```diff
@@ -0,0 +1,54 @@
...
+
+在[应用体积概念](/zh-cn/concepts/size)中你可以深入学习如何创建一个最优化的应用。
+
+### 灵活的架构
+
+由于Tauri使用了web技术，这也意味着几乎所有的前端框架都与Tauri兼容。[前端配置指南](/zh-cn/guides/frontend/)中涵盖了热门前端框架的通用配置；[渲染概念](/zh-cn/concepts/rendering)中讨论了哪些渲染技术（如单页应用SPA和服务端生成SSG）最适合Tauri。
```
Comment (dklassic) [inline src/content/docs/zh-CN/guides/index.mdx]:
```suggestion
由于 Tauri 使用了 web 技术，这也意味着几乎所有的前端框架都与 Tauri 兼容。[前端配置指南](/zh-cn/guides/frontend/)中涵盖了热门前端框架的通用配置；[渲染概念](/zh-cn/concepts/rendering)中讨论了哪些渲染技术（如单页应用 SPA 和服务端生成 SSG）最适合 Tauri。
```
Improve readability.

File: src/content/docs/zh-CN/guides/index.mdx [hunk 14]
```diff
@@ -0,0 +1,54 @@
...
+
+开发者可以在JavaScript中使用 `invoke` 函数实现JavaScript与Rust绑定，而Swift和Kotlin绑定则是作为[Tauri插件](/zh-cn/guides/plugins/)
```
Comment (dklassic) [inline src/content/docs/zh-CN/guides/index.mdx]:
```suggestion
开发者可以在 JavaScript 中使用 `invoke` 函数实现 JavaScript 与 Rust 绑定，而 Swift 和 Kotlin 绑定则是作为 [Tauri 插件](/zh-cn/guides/plugins/)
```
Improve readability.

File: src/content/docs/zh-CN/guides/index.mdx [hunk 15]
```diff
@@ -0,0 +1,54 @@
...
+
+[TAO](https://github.com/tauri-apps/tao) 被用于管理Tauri窗口创建， [WRY](https://github.com/tauri-apps/wry) 则被用于管理webview渲染。这些库由Tauri维护，并且如果需要Tauri没有暴露的更深入的系统集成时可以直接使用。
```
Comment (dklassic) [inline src/content/docs/zh-CN/guides/index.mdx]:
```suggestion
[TAO](https://github.com/tauri-apps/tao) 被用于管理 Tauri 窗口创建，[WRY](https://github.com/tauri-apps/wry) 则被用于管理 webview 渲染。这些库由 Tauri 维护，并且如果需要 Tauri 没有暴露的更深入的系统集成时可以直接使用。
```
Improve readability.

File: src/content/docs/zh-CN/guides/index.mdx [hunk 16]
```diff
@@ -0,0 +1,54 @@
...
+
+除此之外，Tauri还维护了一些插件拓展Tauri核心暴露的系统能力。你可以在[功能及秘诀章节](/zh-cn/features)同时找到官方和社区提供的插件。
```
Comment (dklassic) [inline src/content/docs/zh-CN/guides/index.mdx]:
```suggestion
除此之外，Tauri 还维护了一些插件拓展 Tauri 核心暴露的系统能力。你可以在[功能及秘诀章节](/zh-cn/features)同时找到官方和社区提供的插件。
```
Improve readability.

**Review by dklassic — CHANGES_REQUESTED** [review]
Thanks for contributing! The translation is great with only some formatting suggestions to improve readability. Generally it would be easier to read articles with a mix of both Chinese and English with a space in between Chinese characters (except punctuations) and half sized characters.

**Comment by hangriver** [comment]
@dklassic, thank you for your suggestions! Just to update you, all the suggested changes have been committed. Additionally, as mentioned by @lorenzolewis, the stubs have indeed been removed.

**Comment by dklassic** [comment]
@hangriver nice!
Quick note that some of the formatting suggestions were left untouched, I’m guessing that you probably missed them due to me submitting plenty and GitHub folded them. But if you prefer the original version I’m totally up to discussion.

**Comment by hangriver** [comment]
@dklassic, sorry for my late reply. I have committed the folded suggestions and double-checked the content to make sure all the formatting issues have been corrected.

**Comment by dklassic** [comment]
LGTM!
Thank you for all your work :)

**Pull request merged**